### ISSUE TITLE
Merging multiple files with wildcard

### ISSUE THREAD

**Issue by h3xstream** [microsoft/sarif-sdk#2389]
The multitool has a [merge ](https://github.com/microsoft/sarif-sdk/blob/main/docs/multitool-usage.md) operation. I can't find a working example with the wildcard option.

```
>sarif-multitool merge
SARIF Multitool 2.4.11
c Microsoft Corporation. All rights reserved.
[...]
  <files> (pos. 0)              Required. Files to process (wildcards ? and * allowed).
```

## Test scenario

I have the following files:
```
>dir . /s /b | grep spotbugsSarif.json
[...]\WebGoat\webgoat-container\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\auth-bypass\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\bypass-restrictions\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\challenge\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\chrome-dev-tools\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\cia\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\client-side-filtering\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\cross-site-scripting\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\crypto\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\csrf\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\html-tampering\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\http-basics\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\http-proxies\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\idor\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\insecure-deserialization\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\insecure-login\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\jwt\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\missing-function-ac\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\password-reset\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\path-traversal\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\secure-passwords\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\spoof-cookie\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\sql-injection\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\ssrf\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\vulnerable-components\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\webgoat-introduction\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\webgoat-lesson-template\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\webwolf-introduction\target\spotbugsSarif.json
[...]\WebGoat\webgoat-lessons\xxe\target\spotbugsSarif.json
[...]\WebGoat\webgoat-server\target\spotbugsSarif.json
[...]\WebGoat\webwolf\target\spotbugsSarif.json
```

Running the following command .. Does not seems to grab any files..
```
>sarif-multitool merge */spotbugsSarif.json
Merge completed in 00:00:00.0015308.
```

I also tried few variations..
```
>sarif-multitool merge -p --merge-runs --force */spotbugsSarif.json
Merge completed in 00:00:00.0649805.
```
```
>sarif-multitool merge -p --merge-runs --force */*/spotbugsSarif.json
Merge completed in 00:00:00.0649805.
```

None worked

**Comment by eddynaka** [comment]
@yongyan-gh @shaopeng-gh , can you take a look?

**Comment by h3xstream** [comment]
The tool might be expecting a single directory rather than files: 
https://github.com/microsoft/sarif-sdk/blob/38ad002f2f632fe395fba1a5e96549b8ddb6db14/src/Test.UnitTests.Sarif.Multitool.Library/MergeCommandTests.cs#L53

**Comment by shaopeng-gh** [comment]
Hello, I will take a look.

**Comment by shaopeng-gh** [comment]
@h3xstream Hi I tried in windows below:

`sarif-multitool merge C:\DEL\Input\*.sarif --recurse --output-directory=C:\DEL\Output\ --output-file=MergeResult.sarif --force`

It seems to be able to create the merged .sarif file at the target folder.
I created the folder structure and put a list of .sarif files in C:\DEL\Input\ folder before I ran it.
And the result is C:\DEL\Output\MergeResult.sarif with bigger file size containing the data from each .sarif in the C:\DEL\Input\ folder.

Let us know if with similar parameters works for you.
